c/driver/postgresql/connection.cc

```diff
@@ -844,6 +849,55 @@ AdbcStatusCode PostgresConnection::GetObjects(
   return BatchToArrayStream(&array, &schema, out, error);
 }
 
+AdbcStatusCode PostgresConnection::GetOption(const char* option, char* value,
+                                             size_t* length, struct AdbcError* error) {
+  std::string output;
+  if (std::strcmp(option, ADBC_CONNECTION_OPTION_CURRENT_CATALOG) == 0) {
```

Comment:
I think can simplify this the same way as https://github.com/apache/arrow-adbc/pull/880

Reply:
Hmm, 

> The following functions return parameter values established at connection. These values are fixed for the life of the connection.

It looks like we might not want to use this then, since you can change the current database at runtime

Reply:
I think postgres requires you to disconnect/connect if you want to change databases? When using psql as an example you would have to do `\connect <other_db>` to make that switch

Reply:
Ah! You're right. I'll fix this up then.